Core/include/Acts/EventData/SingleFreeTrackParameters.hpp

```diff
@@ -182,9 +131,9 @@ class SingleFreeParameters {
   ///
   /// @param [in] rhs Object to compare `*this` to
   ///
-  /// @return Boolean value whether the objects can be casted into each other
-  /// and the content of the member variables is the same
-  bool operator==(const SingleFreeParameters& rhs) const {
+  /// @return Boolean value whether the objects can be casted into each
+  /// other and the content of the member variables is the same
+  bool operator==(const SingleFreeTrackParameters& rhs) const {
```

Comment:
Out of curiosity: Why does it work without specifying the class template in the argument?

Reply:
Because this is defined inside the class and the compiler already knows that this is templated. There is no ambiguity here that would need to be resolved.